lxqtgridlayout.cpp

```diff
@@ -363,7 +375,29 @@ void GridLayout::setDirection(GridLayout::Direction value)
         invalidate();
     }
 }
+ 
+/************************************************
+
+************************************************/
+GridLayout::ItemAppendingLocation  GridLayout::itemAppendingLocation() const
+{
+    Q_D(const GridLayout);
+    return d->mItemAppendingLocation;
+}
+
+
+/************************************************
 
+************************************************/
+void GridLayout::setItemAppendingLocation(GridLayout::ItemAppendingLocation value)
+{
+    Q_D(GridLayout);
+     if (d->mItemAppendingLocation != value)
+     {
+         d->mItemAppendingLocation = value;
+         invalidate();
+     }
+}
```

Comment:
Please remove the above methods and add these after `void GridLayout::setStretch(Stretch value)`:

```c++
/************************************************

 ************************************************/
GridLayout::ItemsOrder  GridLayout::itemsOrder() const
{
    Q_D(const GridLayout);
    return d->mItemsOrder;
}


/************************************************

 ************************************************/
void GridLayout::setItemsOrder(GridLayout::ItemsOrder value)
{
    Q_D(GridLayout);
    if (d->mItemsOrder != value)
    {
        d->mItemsOrder = value;
        std::reverse(d->mItems.begin(), d->mItems.end());
        invalidate();
    }
}
```